Ghost_in_the_Shell/client_folder_react_app/ethisim/10

```diff
@@ -0,0 +1,8 @@
++ lint-staged@10.4.0
```

Comment:
It Run linters against staged git files
"It will install and configure husky and lint-staged depending on code quality tools from package.json dependencies so please make sure you install (npm install --save-dev) and configure all code quality tools like Prettier, ESlint prior that."
It was another package that I was told to download alongside husky in order to have Prettier and Eslint work.